### PR TITLE
BUG: append to table from chunks should return table

### PR DESCRIPTION
**Pull request by llllllllll**

### CODE REVIEW

**Comment by cpcloud** [comment]
can you add a test and a bug fix to the whatsnews

**Comment by llllllllll** [comment]
added, merge?

File: docs/source/whatsnew/0.3.3.txt [hunk 1]
```diff
@@ -30,3 +30,5 @@ Bug Fixes
 * PyTables API compatibility when given a integer valued float (:issue:`236`).
 * Default to ``None`` when plucking and a key isn't found (:issue:`228`).
 * Fix gzip dispatching on JSON discovery (:issue:`243`).
+* Fix appending into a sql table from chunks not returning the table.
+  (:issue:`278`).
```
Comment (cpcloud) [inline docs/source/whatsnew/0.3.3.txt]: can you add this note to `0.3.4.txt`

Comment (cpcloud) [inline docs/source/whatsnew/0.3.3.txt]: then good 2 merge

**Pull request merged**